README.md

```diff
@@ -0,0 +1,225 @@
+OSP Director Operator
+=====================
+
+Description
+-----------
+
+
+The OSP Director Operator creates a set of Custom Resource Definitions on top of OpenShift to manage resources normally created by the TripleO's Undercloud. These CRDs are split into two types for hardware provisioning and software configuration.
+
+
+Hardware Provisioning CRDs
+--------------------------
+-openstacknet: Create networks which are used to assign IPs to the vmset and baremetalset resources below
+-openstackvmset: create sets of VMs using OpenShift Virtualization for a specific TripleO role (Controller, Database, NetworkController, etc.)
+-openstackbaremetalset: create sets of baremetal hosts for a specific TripleO role (Compute, Storage, etc.)
+-openstackcontrolplane: A CRD used to create the OpenStack control plane and manage associated openstackvmsets
+-openstackprovisionservers: used to serve custom images for baremetal provisioning with Metal3
+
+Software Configuration CRDs
+---------------------------
+-openstackclient: creates a pod used to run TripleO deployment commands
+
+Installation
+------------
+
+## Prerequisite:
+- OCP 4.6 installed
+- OpenShift Virtualization 2.5
+
+## Install the OSP Director Operator
+The OSP Director Operator is installed and managed via the OLM [Operator Lifecycle Manager](https://github.com/operator-framework/operator-lifecycle-manager). OLM is installed automatically with your OpenShift installation. To obtain the latest OSP Director Operator snapshot you need to create the appropriate CatalogSource, OperatorGroup, and Subscription to drive the installation with OLM:
+
+###Create a CatalogSource (using 'openstack' namespace, and our upstream quay.io tag)
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: osp-director-operator-index
+  namespace: openstack
+spec:
+  sourceType: grpc
+  image: quay.io/openstack-k8s-operators/osp-director-operator-index:0.0.1
+```
+
+###Create an OperatorGroup(using the 'openstack' namespace)
+```yaml
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: "osp-director-operator-group"
+  namespace: openstack
+spec:
+  targetNamespaces:
+  - openstack
+```
+
+###Create a Subscription
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: osp-director-operator-subscription
+  namespace: openstack
+spec:
+  config:
+    env:
+    - name: WATCH_NAMESPACE
+      value: openstack
+  source: osp-director-operator-index
+  sourceNamespace: openstack
+  name: osp-director-operator
+  startingCSV: osp-director-operator.v0.0.1
+  channel: alpha
+```
+
+We have a script to automate the installation here with OLM for a specific tag: [script to automate the installation](https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/scripts/deploy-with-olm.sh)
```

Comment:
I need to fix the referenced script here.  It lacks the `openshift-machine-api` namespace in the `WATCH_NAMESPACE` env config in the generated subscription.

Reply:
would it make sense to fix all these at once in a follow up patch? Alternately I suppose I can add them all here now in this README patch

Reply:
Sounds good